commercial/app/views/books/booksStandardV2.scala.html

```diff
@@ -2,8 +2,30 @@
   omnitureId: Option[String],
   clickMacro: Option[String])(implicit request: RequestHeader)
 
-@containerWrapper(Seq(), "")(HtmlFormat.empty)(HtmlFormat.empty)(HtmlFormat.empty)(HtmlFormat.empty) {
+@defining(("2_0", "2014-10-15")) { case (version, date) =>
```

Comment:
Is there any value in this versioning information?  I doubt it will ever be updated.